pkg/proxy/iptables/proxier.go

```diff
@@ -516,6 +516,7 @@ func (proxier *Proxier) Sync() {
 func (proxier *Proxier) SyncLoop() {
 	t := time.NewTicker(proxier.syncPeriod)
 	defer t.Stop()
+	registerMetrics()
```

Comment:
I didn't want to register those if not in iptables mode. But we have if there anyway.

Will move.